carma_cooperative_perception/src/multiple_object_tracker_component.cpp

```diff
@@ -482,10 +580,16 @@ auto MultipleObjectTrackerNode::execute_pipeline() -> void
   temporally_align_detections(detections_, current_time);
 
   const auto predicted_tracks{predict_track_states(track_manager_.get_all_tracks(), current_time)};
-  const auto scores{
-    mot::score_tracks_and_detections(predicted_tracks, detections_, mot::euclidean_distance_score)};
+  auto scores{
+    mot::score_tracks_and_detections(predicted_tracks, detections_, SemanticDistance2dScore{})};
+
+  // This pruning distance is an arbitrarily-chosen heuristic. It is working well for our
+  // current purposes, but there's no reason it couldn't be restricted or loosened.
+  mot::prune_track_and_detection_scores_if(scores, [](const auto & score) { return score > 5.0; });
```

Comment:
Maybe as a future improvement we can make this score configurable.

Reply:
Not under this PR. Just asking if that makes sense to you.

Reply:
Yeah that makes sense. Will put in the TODO queue for future work.